src/routes/search.ts

```diff
@@ -10,12 +10,7 @@ export default async function place(fastify: FastifyInstance) {
   // returns search results dropdown
   fastify.post('/search', async (req, resp) => {
     const queryParams: any = req.query;
-    const {
-      op,
-      place_id: placeId,
-      type,
-      prefix: dataPrefix
-    } = queryParams;
+    const { op, place_id: placeId, type, prefix: dataPrefix } = queryParams;
```

Comment:
i'm unclear on when you like to consolidate onto a single line and when you like to split into multiple lines. like src/routes/manage-hierarchy.ts line 47 is basiaclly the exact opposite change as this

Reply:
vscode auto-formated the files